### DISCUSSION TITLE
New Theme - Covardex Detailed

### DISCUSSION THREAD

**Discussion by phfleitao** [JanDeDobbeleer/oh-my-posh#4665] [Themes]
Hey everyone! I want to share my theme with you all.
I just need to open a discussion like this one here or i need to do something else?

![covardex-detailed](https://github.com/JanDeDobbeleer/oh-my-posh/assets/48184805/fb2ce019-b703-490c-9daa-aac16015a0a0)

Theme Name : covardex-detailed
Source code : https://github.com/phfleitao/oh-my-posh-themes/tree/main/covardex-detailed

**Comment by JanDeDobbeleer** [comment]
You can also share it on our discord!

**Reply by phfleitao** [reply]
Ok, I'll do that. thanks!

**Reply by JanDeDobbeleer** [reply]
@phfleitao it's right [here](https://discord.com/channels/1023597603331526656/1055533233309233252) btw
